assets/app/views/browse/deployment.html

```diff
@@ -91,9 +91,19 @@
                       name="deploymentConfigName"
                       context="projectContext"
                       options="logOptions"
-                      status="deployment | deploymentStatus"
-                      time-start="deployment.metadata.creationTimestamp | date : 'short'"
                       run="logCanRun">
+                      <div row mobile="column" tablet="column">
+                        <div>Deployment status:&nbsp;</div>
```

Comment:
I'm not sure deployment status is guaranteed to be set. Maybe an `ng-if-start="deployment | deploymentStatus"` here and the end on the div below.